guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/editors/repository/clone/CloneRepositoryView.java

```diff
@@ -0,0 +1,87 @@
+package org.guvnor.structure.client.editors.repository.clone;
```

Comment:
Whilst we're trying to improve out code quality (and test coverage) JavaDocs are always nice to add *now* rather than *later*.